### PR TITLE
DS-3808_editor_config

### PR DESCRIPTION
**Pull request by MatiasRoy-Rodriguez**
feat: config for .editorconfig

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+end_of_line = lf
+charset = utf-8
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.php]
+indent_style = space
+indent_size = 4
```
Comment (andresmoschini) [inline .editorconfig]: These settings are redundant

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+charset = utf-8
+indent_style = space
+indent_size = 4
+insert_final_newline = true
```
Comment (andresmoschini) [inline .editorconfig]: This rule is being broken here: 
image

**Comment by andresmoschini** [comment]
Nice initiative! it is great to document the code convention with editorconfig.

We are already using it in some other Doppler projects, you can see an example here: https://github.com/FromDoppler/doppler-editors-webapp/blob/main/.editorconfig

And you can see how we enforce the rules here: https://github.com/FromDoppler/doppler-editors-webapp/blob/main/package.json#L39-L40

**Comment by MatiasRoy-Rodriguez** [comment]
> Nice initiative! it is great to document the code convention with editorconfig.
> 
> We are already using it in some other Doppler projects, you can see an example here: https://github.com/FromDoppler/doppler-editors-webapp/blob/main/.editorconfig
> 
> And you can see how we enforce the rules here: https://github.com/FromDoppler/doppler-editors-webapp/blob/main/package.json#L39-L40

Great, thanks @andresmoschini for the examples they will be useful for all our repos 😄 !

**Review by hcardoso81 — APPROVED** [review]
good!

**Pull request merged**